index.html

```diff
@@ -1,300 +1,444 @@
-                              </svg></a>
-                          </div>
-                    </div>
-                    <a href="/" class="resume">Get my resume</a>
-        
-                </div>
-                <div class="secseg">
-                    
-                  <ul class="language"><li><b>Languages</b></li><li><svg class="grid2" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
-                      <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z" fill="#5E6C84"/>
-                    </svg></li></ul>
-                    <ul>
-                      <li><img src="asset/images/javascriptlogo.png" alt="lang-logo"><span>Javascript</span></li>
-                      <li><img src="asset/images/htmllogo.png" alt="lang-logo"><span>HTML</span></li>
-                      <li><img src="asset/images/csslogo.png" alt="lang-logo"><span>CSS</span></li>
-                  </ul>
-                  <ul class="work"><li><b>Frameworks</b></li>
-                      <li><svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
-                          <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
-                        </svg></li>
-                  </ul>
-                  <hr>
-                  <ul class="work"><li><b>Skills</b></li>
-                      <li><b><svg xmlns="http://www.w3.org/2000/svg" width="24" height="25" viewBox="0 0 24 25" fill="none">
-                          <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
-                        </svg></b></li>
-                  </ul>
-                  <hr>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Deskdop version</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
- [ ] Your stying is not reflecting on your index file. This is because the `CSS` file path here is not the same as the name of the `CSS` style sheet name of your project.
- Kindly add the missing `s`, and please look at the design. When I added the `s` and ran your code locally, there were still lots of chnages to be made. Please work on it before sending it for another review  :

### View:
![image](https://github.com/okayboy12/Portfolio-setup-and-mobile-first/assets/50990925/328220fd-3a82-4265-bf52-3e0e2705e1ad)